qrcode_app/lib/data/datasource/seeds_api_datasource.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:convert';
+
+import 'package:qrcode_app/data/models/seed_model.dart';
+import 'package:qrcode_app/domain/entities/seed.dart';
+import 'package:http/http.dart' show Client;
+
+class SeedApiDatasource {
+  static const URL = 'http://10.0.0.172:8000';
+
+  static Future<Seed> getSeed(Client client) async {
+    final response =
+        await client.get(Uri.https('seed-server.herokuapp.com', '/seed'));
+
+    if (response.statusCode == 200) {
+      return SeedModel().fromJson(json.decode(response.body));
```

Comment:
**chore**: When using a mapper you would use that here.
`return mapper.toSeed(SeedDto.fromJson(json.decode(response.body)));`
mapper would be an instance member of this datasource class.
`final mapper = SeedMapper();`